terraform-tests/sqs_test.go

```diff
@@ -127,31 +135,10 @@ var _ = Describe("SQS", Label("SQS-terraform"), Ordered, func() {
 		})
 	})
 
-	Context("Standard Queue", func() {
```

Comment:
I removed this test because although the description is unique, the actual test is a duplicate of one above.